rosmon_core/src/launch/launch_config.cpp

```diff
@@ -276,7 +281,7 @@ void LaunchConfig::parseNode(TiXmlElement* element, ParseContext ctx)
 	const char* type = element->Attribute("type");
 	const char* args = element->Attribute("args");
 	const char* ns = element->Attribute("ns");
-	const char* respawn = element->Attribute("respawn");
+        const char* respawn = element->Attribute("respawn");
```

Comment:
this repo uses tabs instead of spaces, please use tabs for this external repo to preserve formatting